app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  attachment = GovukPublishingComponents::Presenters::Attachment.new(attachment)
+  target ||= nil
+  hide_help_text ||= false
+  attributes = []
+
+  if attachment.content_type_abbr
+    content = tag.abbr(attachment.content_type_abbr,
+                       class: "gem-c-attachment__abbr",
+                       title: attachment.readable_content_type)
+    attributes << tag.span(content)
```

Comment:
I think we should add similar classes to the ones we have for attachment link: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_attachment_link.html.erb#L18

Mainly because it'd be such a pain to do any styling without the classes once these components are embedded in content HTML.

Reply:
I noticed that `gem-c-attachment-link__attribute` isn't in attachment-link.scss.  Should we be adding classes with no styling to the markup?

Reply:
I'd suggest so, since we can't add the classes later once this is embedded in a documents HTML.

Reply:
I've added these classes in a separate commit just to be clear about why they don't have any styling yet.